doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,384 @@
+
+|PXC| currently supports replication only for tables
+that use a transactional storage engine (XtraDB or InnoDB).
+To ensure data consistency,
+the following statements should not be allowed for tables
+that use a non-transactional storage engine (MyISAM, MEMORY, CSV, etc.):
+
+* Data manipulation statements that perform writing to table
+  (for example, ``INSERT``, ``UPDATE``, ``DELETE``, etc.)
+
+* The following administrative statements:
+  ``CHECK``, ``OPTIMIZE``, ``REPAIR``, and ``ANALYZE``
+
+* ``TRUNCATE TABLE`` and ``ALTER TABLE``
+
+Depending on the strict mode selected,
+the following happens:
+
+``DISABLED``
+
+ At startup, no validation is performed.
+
+ At runtime, all operations are permitted.
+
+``PERMISSIVE``
+
+ At startup, no validation is perfromed.
+
+ At runtime, all operations are permitted,
+ but a warning is logged when an undesirable operation
+ is performed on an unsupported table.
+
+``ENFORCING`` or ``MASTER``
+
+ At startup, no validation is performed.
+
+ At runtime, any undesirable operation performed on an unsupported table
+ is denied and an error is logged.
+
+.. note:: Unsupported tables can be converted to use a supported storage engine.
+
+MyISAM replication
+------------------
+
+|PXC| provides experimental support for replication of tables
+that use the MyISAM storage engine.
+Due to the non-transactional nature of MyISAM,
+it is not likely to ever be fully supported in |PXC|.
+
+MyISAM replication is controlled
+using the :variable:`wsrep_replicate_myisam` variable,
+which is set to ``OFF`` by default.
+Due to its unreliability, MyISAM replication should not be enabled
+if you want to ensure data consistency.
+
+Depending on the strict mode selected,
+the following happens:
+
+``DISABLED``
+
+ At startup, no validation is performed.
+
+ At runtime, you can set :variable:`wsrep_replicate_myisam` to any value.
+
+``PERMISSIVE``
+
+ At startup, if :variable:`wsrep_replicate_myisam` is set to ``ON``,
+ a warning is logged and startup continues.
+
+ At runtime, it is permitted to change :variable:`wsrep_replicate_myisam`
+ to any value, but if you set it to ``ON``, a warning is logged.
+
+``ENFORCING`` or ``MASTER``
+
+ At startup, if :variable:`wsrep_replicate_myisam` is set to ``ON``,
+ an error is logged and startup is aborted.
+
+ At runtime, any attempt to change :variable:`wsrep_replicate_myisam`
+ to ``ON`` fails and an error is logged.
+
+.. note:: The :variable:`wsrep_replicate_myisam` variable controls
+   *replication* for MyISAM tables,
+   and this validation only checks whether it is allowed.
+   Undesirable operations for MyISAM tables
+   are restricted using the :ref:`storage-engine` validation.
+
+Binary log format
+-----------------
+
+|PXC| supports only the default row-based binary logging format.
+This validation checks the value of the |binlog_format|_ variable
+at session level, because it cannot be changed globally.
+
+Depending on the strict mode selected,
+the following happens:
```

Comment:
Allowing setting of binlog_format at startup will set global variable something that is not allowed.

Check this

DISABLED:
At runtime, setting binlog_format to STATEMENT/MIXED/ROW is allowed.
PERMISSIVE:
At runtime, setting binlog_format to STATEMENT/MIXED logs a warning. New setting is accepted. (Setting it to ROW is welcome operation and doesn't result in a warning)
ENFORCING/MASTER:
At runtime, setting binlog_format to STATEMENT/MIXED logs an error. SET statement fails. (Setting it to ROW is welcome operation and doesn't result in an error).